app/assets/stylesheets/application.scss

```diff
@@ -54,3 +54,8 @@
 
 // components
 @import 'govuk_publishing_components/all_components';
+
+.gem-c-radio__input {
```

Comment:
Why are these styles here? It looks like the size of these elements is already defined in the component (where it should be) and disabling them [doesn't change the appearance](https://govuk-frontend-app-pr-1509.herokuapp.com/sold-bought-vehicle/y)

Reply:
Yes! Wanted to ask you about this. The `em` thing doesn't seem to work in this application so the clickable area is about half the size:

![screen shot 2018-05-09 at 09 58 42](https://user-images.githubusercontent.com/233676/39805809-c214a5ea-536f-11e8-8be1-4456e6e8cc06.png)

Reply:
Ah, I see the problem now. Seems to be that the component is inadvertently relying upon some core styles for inputs, which includes font-size: inherit. As a test, if you replace these two lines with that, the input comes out at the right size. Suggest we make an adjustment to the component to define an explicit font size so this works in isolation - might be able to move the font size declaration for the label higher up so it applies to the whole component.

As a side point, I'm surprised that the radio button works this way. I would have thought it would be much easier to simply make the label full width and play with left padding and positioning of the circle, so even though it looks like you're clicking on the radio button you'd in fact always be clicking on the label. But that would involve rebuilding the component, which is way outside of this scope.